### PR TITLE
[Doc] Add site2/website/versioned_docs/version-2.3.1/getting-started-docker.md

### PR DESCRIPTION
**Pull request by Jennifer88huang**
### Motivation
I've corrected the command in this file in #4121, the change does not effect in version 2.3.1 document, since I didn't move the updated file to the site2/website/versioned_docs/version-2.3.1 directory.  

### Modifications
Copy the updated file `getting-started-docker.md` to the site2/website/versioned_docs/version-2.3.1 directory.

### CODE REVIEW

**Comment by Jennifer88huang** [comment]
run integration tests

**Comment by Jennifer88huang** [comment]
run java8 tests

**Comment by Jennifer88huang** [comment]
Run CPP Tests

File: site2/website/versioned_docs/version-2.3.1/getting-started-docker.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+id: standalone-docker
```
Comment (murong00) [inline site2/website/versioned_docs/version-2.3.1/getting-started-docker.md]: the `id` should be `version-2.3.1-standalone-docker`

File: site2/website/versioned_docs/version-2.3.1/getting-started-docker.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+title: Start a standalone cluster with Docker
+sidebar_label: Pulsar in Docker
+---
```
Comment (murong00) [inline site2/website/versioned_docs/version-2.3.1/getting-started-docker.md]: `original_id` should be added.

**Comment by Jennifer88huang** [comment]
@murong00 Thank you very much for your kinder reminder, I've update the file as per your advice.

**Comment by sijie** [comment]
run integration tests

**Pull request merged**